gradle-nonnull-plugin/plugin/src/test/groovy/com/novoda/gradle/nonnull/AndroidProjectIntegrationTest.groovy

```diff
@@ -0,0 +1,62 @@
+package com.novoda.gradle.nonnull
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.internal.DefaultGradleRunner
+import org.junit.ClassRule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+import static com.google.common.truth.Truth.assertThat
+
+public class AndroidProjectIntegrationTest {
+
+    @ClassRule
+    public static final ProjectRule PROJECT = new ProjectRule()
+
+    @Test
+    public void shouldGeneratePackageAnnotation() {
+        PROJECT.generatedSrcDirs.each {
+            def file = new File(it, 'package-info.java')
```

Comment:
`nit:` I would suggest to try to avoid `def` unless properly needed. It's easier for the IDE to work out autocompletion & refactoring when type is explicit. It also minimise mistakes when code gets shuffled around.